res/res_pjsip_outbound_registration.c

```diff
@@ -1548,6 +1575,7 @@ static struct sip_outbound_registration_state *sip_outbound_registration_state_a
 	state->client_state->transport_name = ast_strdup(registration->transport);
 	state->client_state->registration_name =
 		ast_strdup(ast_sorcery_object_get_id(registration));
+	state->client_state->user_agent = ast_strdup(registration->user_agent);
```

Comment:
You'll need to add an ast_free() for this in sip_outbound_registration_client_state_destroy().

Reply:
Fixed